src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -710,6 +710,9 @@ def _tulsi_sources_aspect(target, ctx):
                         _collect_bundle_imports(rule_attr))
 
     copts_attr = _get_opt_attr(rule_attr, "copts")
+    if copts_attr:
```

Comment:
Please add a comment here mentioning why the expansion is needed (an example would be good)

Reply:
I added a detailed comment with an example. 
@ob has an example in https://github.com/bazelbuild/bazel/issues/7083 in the form of a `repro.sh` to reproduce this issue. Does that help for integration testing?